### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by MohamedOmarii**
related to #22

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const tape = require('tape');
+const request = require('supertest');
+const server = require('../server.js');
+
+tape('test tape', (t) => {
```
Comment (mattlub) [inline tests/tests.js]: you can remove this test.

File: tests/tests.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  t.ok(true, 'its working');
+  t.end();
+});
+
+tape('home route test: GET request to /', t => {
+  request(server)
+    .get('/')
+    .end((err, res) => {
```
Comment (mattlub) [inline tests/tests.js]: if you want you can do 
```js
.get('/')
.expect(200)
.end((err, res) => {
  // there will be an error here if it is not a 200
})

**Comment by mattlub** [comment]
also add a pre-commit hook?

File: tests/tests.js (hunk 1 above)
Comment (MohamedOmarii) [inline tests/tests.js]: hahaha yeah

File: tests/tests.js (hunk 2 above)
Comment (MohamedOmarii) [inline tests/tests.js]: :+1:

Comment (MohamedOmarii) [inline tests/tests.js]: in both it working hahaha

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 6.10.0
+
+install:
+  -npm install
+
+script:
+  -npm run coverage
```
Comment (mattlub) [inline .travis.yml]: this should be test

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 6.10.0
```
Comment (mattlub) [inline .travis.yml]: should be 6.11

Comment (MohamedOmarii) [inline .travis.yml]: ok :+1:

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+install:
+  - npm install
+
+script:
+  - npm run coverage
```
Comment (mattlub) [inline .travis.yml]: should be npm test

File: .travis.yml (hunk 4 above)
Comment (MohamedOmarii) [inline .travis.yml]: not working !!

**Pull request merged**